Nclient.html

```diff
@@ -23,56 +23,435 @@
+      						<option value="9">FL</option>
+      						<option value="10">GA</option>
+      						<option value="11">HI</option>
+      						<option value="12">ID</option>
+      						<option value="13">IL</option>
+      						<option value="14">IN</option>
+      						<option value="15">IA</option>
+      						<option value="16">KS</option>
+      						<option value="17">KY</option>
+      						<option value="18">LA</option>
+      						<option value="19">ME</option>
+      						<option value="20">MD</option>
+      						<option value="21">MA</option>
+      						<option value="22">MI</option>
+      						<option value="23">MN</option>
+      						<option value="24">MS</option>
+      						<option value="25">MO</option>
+      						<option value="26">MT</option>
+      						<option value="27">NE</option>
+      						<option value="28">NV</option>
+      						<option value="29">NH</option>
+      						<option value="30">NJ</option>
+      						<option value="31">NM</option>
+      						<option value="32">NY</option>
+      						<option value="33">NC</option>
+      						<option value="34">ND</option>
+      						<option value="35">OH</option>
+      						<option value="36">OK</option>
+      						<option value="37">OR</option>
+      						<option value="38">PA</option>
+      						<option value="39">RI</option>
+      						<option value="40">SC</option>
+      						<option value="41">SD</option>
+      						<option value="42">TN</option>
+      						<option value="43">TX</option>
+      						<option value="44">UT</option>
+      						<option value="45">VT</option>
+      						<option value="46">VA</option>
+      						<option value="47">WA</option>
+      						<option value="48">WV</option>
+      						<option value="49">WI</option>
+      						<option value="50">WY</option>
+    						</select>
+    						<label>State</label>
+						</div>
+						<div class="input-field col s3">
+							<input type="text" name="def_zip">
+							<label for="def_zip">Zip Code</label>	
+						</div>
+					</div>
+
+					<div class="row">
+						<div class="col s3">
+							<a class="waves-effect waves-light btn lime accent-3">Add</a>
+							<span>Another Address</span>
 
+							
+						</div>
+						
 					</div>
-			</form>
+				</form>
+			</div>
 		</div>
-	</div>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 
 
 <script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/js/materialize.min.js"></script>
 
 	<script>
+
 		$(document).ready(function() {
     	$('select').material_select();
   	});
+		window.modalName = '';
+
+		$('.js-change-modal').on('click', function() {
+  			var _this = $(this);
+  			window.modalName = _this.attr('data-modal');
+	});
+		
+		$(document).ready(function(){
+    	$('.modal').modal();
+  	});
+		$(document).ready(function() {
+    	$('select').material_select();
```

Comment:
you dont want to have 2 `document.ready`'s here. Move `$('select').material_select();` into your first `document.ready` (under line `450`)

Reply:
oh cool ok